### PR TITLE
add: intro to datatypes

### PR DESCRIPTION
**Pull request by nizolnier**

### CODE REVIEW

File: src/intro-to-c/datatypes/datatypes-intro.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    ```c
+    long double e = 1.0000000000000000001
+    ```
+- **char:** stores a single character (1 byte)
+    ```c
+    char f = '@'
+    ```
+- **string:** an array of char's that allows you to form words
+    ```c
+    char[] g = 'hello'
```
Comment (jerrettl) [inline src/intro-to-c/datatypes/datatypes-intro.md]: Got a small fix for you! :)
```suggestion
    char[] g = "hello"
```

File: src/intro-to-c/datatypes/datatypes-intro.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+geometry: margin=1in
+---
+
```
Comment (jerrettl) [inline src/intro-to-c/datatypes/datatypes-intro.md]:
```suggestion
```

File: src/intro-to-c/datatypes/datatypes-intro.md (hunk 1 above)
Comment (nizolnier) [reply src/intro-to-c/datatypes/datatypes-intro.md]: thanks!

**Review by jerrettl — APPROVED** [review]
😎

**Comment by jerrettl** [comment]
You can merge whenever you like. Your PR, your merge :)

**Pull request merged**